templates/repo/home.tmpl

```diff
@@ -9,29 +9,18 @@
 		</p>
 		<div class="ui secondary menu">
 			{{if .PullRequestCtx.Allowed}}
+			{{if ne .BranchName .BaseRepo.DefaultBranch}}
 				<div class="fitted item">
 					<a href="{{.BaseRepo.Link}}/compare/{{.BaseRepo.DefaultBranch}}...{{if .SignedUser.HasForkedRepo .BaseRepo.ID }}{{.SignedUser.Name}}:{{end}}{{.BranchName}}">
-						<button class="ui green small button"><i class="octicon octicon-git-compare"></i></button>
+						<button class="ui green tiny compact button"><i class="octicon octicon-git-compare"></i></button>
 					</a>
 				</div>
 			{{end}}
+			{{end}}
 			{{template "repo/branch_dropdown" .}}
-			<div class="fitted item">
-				<span class="ui breadcrumb">
-					<a class="section" href="{{.RepoLink}}/src/{{EscapePound .BranchName}}">{{EllipsisString .Repository.Name 30}}</a>
-					{{ $n := len .TreeNames}}
-					{{ $l := Subtract $n 1}}
-					{{range $i, $v := .TreeNames}}
-						<span class="divider"> / </span>
-						{{if eq $i $l}}
-							<span class="active section">{{EllipsisString $v 30}}</span>
-						{{else}}
-							{{ $p := index $.Paths $i}}
-							<span class="section"><a href="{{EscapePound $.BranchLink}}/{{EscapePound $p}}">{{EllipsisString $v 30}}</a></span>
-						{{end}}
-					{{end}}
-				</span>
-			</div>
+			{{ $n := len .TreeNames}}
+			{{ $l := Subtract $n 1}}
+			<div class="fitted item"><span class="ui breadcrumb repo-path"><a class="section" href="{{.RepoLink}}/src/{{EscapePound .BranchName}}">{{EllipsisString .Repository.Name 30}}</a>{{range $i, $v := .TreeNames}}<span class="divider">/</span>{{if eq $i $l}}<span class="active section">{{EllipsisString $v 30}}</span>{{else}}{{ $p := index $.Paths $i}}<span class="section"><a href="{{EscapePound $.BranchLink}}/{{EscapePound $p}}">{{EllipsisString $v 30}}</a></span>{{end}}{{end}}</span></div>
```

Comment:
Unfortunately not, if we want the path to be copyable as-is. If I add any indentation to this block, a path like `someproject/somefile` will copy as `someproject / somefile` or to the clipboard. Even adding indentation to the wrapping `<div>` will result in a trailing whitespace being copied.

As mentioned, a proper fix would be to minify the served HTML, but that has its downsides too, like decreased performance because of the minifier having to run before serving any HTML.